app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] This module requires that `Post` model has method that updates the posts counter for a user. This should be similar to the example below:

```ruby
# app/models/post.rb
...

def update_post_counter
  author.increment!(:posts_counter)
end
```

Reply:
well resolved! ✔️